hack/generator/Makefile

```diff
@@ -17,11 +17,11 @@ build: tidy fmt vet lint ; $(info $(M) buiding ./bin/$(APP))
 
 .PHONY: lint
 lint: $(GOLANGCI_LINT) ; $(info $(M) running golangci configured linters…) ## Lint codebase
-	$(Q) $(GOLANGCI_LINT) run -v --timeout 5m
+	$(Q) $(GOLANGCI_LINT) run -v
```

Comment:
Is there a timeout on the CI pipeline that will catch long running execution anyway?

Reply:
It's set in the config file I just forgot to remove it here